src/autocomplete/src/Autocomplete.js

```diff
@@ -15,18 +15,61 @@ const autocompleteItemRenderer = props => <AutocompleteItem {...props} />
 // https://github.com/paypal/downshift/issues/164
 export default class Autocomplete extends PureComponent {
   static propTypes = {
+    /**
+     * This prop can be either a string or a Node.
+     * It will provide a title for the items
+     */
     title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    /**
+     * An array of items to be used as options for the select
+     */
     items: PropTypes.array.isRequired,
+    /**
+     * The selected Item to be shown on the autocomplete
+     */
     selectedItem: PropTypes.any,
+    /**
+     * The selected item to be selected & shown by default on the autocomplete
+     */
     defaultSelectedItem: PropTypes.any,
+    /**
+     * In case the array of items is not an array of strings, this function is used on each item to return the string that will be shown on the filter
+     */
     itemToString: PropTypes.func.isRequired,
+    /**
+     * Function that will render the 'filter' component.
+     */
     children: PropTypes.func.isRequired,
+    /**
+     * Proptype to set the lits items size beforehand for the virtual list to render properly.
```

Comment:
The height of each item. Because the list is virtualized this is required beforehand.